From: Gerald Pfeifer <gerald@pfeifer.com>
Subject: [PATCH] configure: Diagnose if NetAPI is not present
Message-Id: <20211123200716.3C55933F27@hamza.pair.com>
Date: Tue, 23 Nov 2021 21:07:18 +0100 (CET)

For most other configure options we are warning when some dependency
is not present, in particular when it's been explicitly requested.
Also do this for Samba NetAPI now.

Signed-off-by: Gerald Pfeifer <gerald@pfeifer.com>
---
 configure.ac | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/configure.ac b/configure.ac
index 1c716b4ec00..535eb5d8793 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1722,6 +1722,10 @@ then
     WINE_PACKAGE_FLAGS(NETAPI,[netapi],,,,
       [WINE_CHECK_SONAME(netapi,libnetapi_init,,[AC_DEFINE_UNQUOTED(SONAME_LIBNETAPI,["libnetapi.$LIBEXT"])],[$NETAPI_LIBS])])
 fi
+WINE_NOTICE_WITH(netapi,[test "x$ac_cv_lib_netapi" != xyes],
+                 [libnetapi not found, Samba NetAPI won't be supported.],
+                 [enable_netapi])
+
 
 dnl **** Check for any sound system ****
 if test "x$enable_winealsa_drv$enable_winecoreaudio_drv$enable_winepulse_drv$enable_wineoss_drv$enable_wineandroid_drv" = xnonononono -a \

-- 
2.32.0

